go/engine/common.go

```diff
@@ -61,9 +64,15 @@ func findPaperKeys(ctx *Context, g *libkb.GlobalContext, me *libkb.User) (*keypa
 		g.Log.Debug("paper version mismatch: generated paper key version = %d, libkb version = %d", version, libkb.PaperKeyVersion)
 		return nil, libkb.KeyVersionError{}
 	}
+	if !paperPhrase.ValidWords() {
+		g.Log.Debug("paper phrase has invalid word(s) in it")
+		return nil, libkb.PassphraseError{Msg: "invalid word(s) in paper key phrase"}
```

Comment:
It's nearly impossible to get this error because the checker on paper key phrase already checks this.  I just left this in for safety.